<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
		<title>iDEC NEFU_China 2022</title>
		<link rel="stylesheet" type="text/css" href="../../css/main_cssstyle.css">
		<style>
			header {
				background: url("../../static/top/experiment.png");
				min-height: 50%;
				background-repeat: no-repeat;
				background-size: cover;
			}

			.page-title {
				padding-top: 10em;
				padding-bottom: 1.5em;
			}

			/* footer的样式 */
			.contact-icon {
				display: flex;
				align-items: center;
				/*font-size: 3em;*/
				color: white;
			}

			.contact-icon a {
				color: white;
			}

			.contact-icon i {
				font-size: 3em;
			}
		</style>
	</head>
	<body>
		<a class="skip-main" href="#slide1-chapter1">Skip to main content</a>
		<header>
			<nav id="normal-nav">
				<!---- 常规导航栏Navigation bar (normal) -->
				<div class="logo">
					<img style="max-height: 22em" id="nav-logo" src="../../static/logo.png"
						alt="This is our logo, but if you click it, you will return home">
				</div>
				<div class="chapters">
					<div class="dropdown-menu cool-link">
						<a class="chapter-nav " href="https://idec-teams.github.io/2022_NEFU_China/">
							<div class="chapter-content">
								<div class="chapter-title active ">
									Home
								</div>
							</div>
						</a>
					</div>
					<div class="dropdown-menu cool-link">
						<a class="chapter-nav " href="#">
							<div class="chapter-content">
								<div class="chapter-title active ">
									Team
								</div>
								<div class="chapter-icon">
									<i class="fas fa-sort-down"></i>
								</div>
							</div>
						</a>
						<div class="dropdown-content">
							<a href="../team/members.html" class="dropdown-page cool-link2">
								Members
							</a>
							<a href="../team/attributions.html" class="dropdown-page cool-link2">
								Attributions
							</a>
						</div>
					</div>
					<div class="dropdown-menu cool-link">
						<a class="chapter-nav " href="#">
							<div class="chapter-content">
								<div class="chapter-title active ">
									Project
								</div>
								<div class="chapter-icon">
									<i class="fas fa-sort-down"></i>
								</div>
							</div>
						</a>
						<div class="dropdown-content">
							<a href="../project/background.html" class="dropdown-page cool-link2">
								Background
							</a>
							<a href="../project/design.html" class="dropdown-page cool-link2">
								Design
							</a>
							<a href="../project/results.html" class="dropdown-page cool-link2">
								Results
							</a>
							<a href="../../static/report.pdf" class="dropdown-page cool-link2">
								Report
							</a>
							<a href="../../static/supplementary information.pdf" class="dropdown-page cool-link2">
								Supplementary information
							</a>
						</div>
					</div>
					<div class="dropdown-menu cool-link">
						<a class="chapter-nav " href="#">
							<div class="chapter-content">
								<div class="chapter-title active ">
									Documentation
								</div>
								<div class="chapter-icon">
									<i class="fas fa-sort-down"></i>
								</div>
							</div>
						</a>
						<div class="dropdown-content">
							<a href="../documentation/experiments.html" class="dropdown-page cool-link2">
								Experiments
							</a>
							<a href="../documentation/notebook.html" class="dropdown-page cool-link2">
								Notebook
							</a>
							<a href="../documentation/safety.html" class="dropdown-page cool-link2">
								Safety
							</a>
							<a href="../project/protocol.html" class="dropdown-page cool-link2">
								Protocol
							</a>
						</div>
					</div>
				</div>
			</nav>
			<!------------ 页面标题PAGE TITLE -------------------->
			<div class="page-title">
				<div class="page-title-content">
					<h1>EXPERIMENTS</h1>
					<!---Page title -->
				</div>
			</div>
		</header>
		<div class="grid-normal">
			<!--------------Introduction/Summary--------------------->
			<div class="introduction grid-normal">
			</div>
		</div>
		<div style="display:none" id="marker" class="triangle-down"> </div>
		<!------------------ BUTTONS ---------------------------->
		<div class="buttons">
		</div>
		<div class="sections-container">
			<aside class="sticky">
				<div id="sub-nav" class="sub-navigation-content">
					<div class="sub-navigation-chapter cool-link2">
						<a class="main-nav-link" linkedidchapter="slide1-chapter1">Designing a dual selection system<p>
						</a>
						<!------Chapter Two -->
					</div>
					<div class="sub-navigation-chapter cool-link2">
						<a class="main-nav-link" linkedidchapter="slide1-chapter2">Directed evolution of PobR</a>
						<!------Chapter Three -->
					</div>
					<div class="sub-navigation-chapter cool-link2">
						<a class="main-nav-link" linkedidchapter="slide1-chapter3">Fluorescence assay and screening of
							the PobR mutant library</a>
						<!------Chapter Three -->
					</div>
					<div class="sub-navigation-chapter cool-link2">
						<a class="main-nav-link" linkedidchapter="slide1-chapter4">The specificity and detection range
							of PobR mutants responsive to new ligands </a>
						<!------Chapter Three -->
					</div>
					<div class="sub-navigation-chapter cool-link2">
						<a class="main-nav-link" linkedidchapter="slide1-chapter5">Modeling and docking</a>
						<!------Chapter Three -->
					</div>

					<!--            返回顶部箭头-->
					<div id="to-top-arrow">
						<a title="Back to top" href="#"><i class="fas fa-chevron-circle-up"></i></a>
						<!------Chapter Three -->
					</div>
				</div>
			</aside>
			<div class="carousel">
				<div class="slider">
					<!------------------------------ section 1------------------------------------>
					<section class="slide-sections">
						<div class="main-body">
							<div class="main-body-content">
								<div id="slide1-chapter1">
									<h2>Designing a dual selection system</h2>
									<p>We constructed a dual selection system based on plasmid
										gYB2a-pobRWT-mCherry-codA-cmr. This plasmid mainly contains three parts:</p>
									<p>1. The <i>pobR</i> coding sequence (CDS) which can express PobR protein;</p>
									<p>2. The cytosine deaminase(<i>codA</i>) gene which can express CDase protein;</p>
									<p>3. An engineered operon consisting of the red fluorescent protein(<i>mCherry</i>)
										and the chloramphenicol(Cm) resistant gene (<i>cmr</i>).</p>
									<p>How this plasmid acts as a dual selection system is as follows:</p>
									<p>1. Negative selection: Been activated by PobR<sup>WT</sup>, P<sub>pobA</sub>
										activates the expression of CDase which is available for conversion of
										exogenously added 5-FC to 5-FU, causing cell death;</p>
									<p>2. Positive selection: Without no additional 5-FC, adding chloramphenicol and an
										aromatic compound to cultivate <i>E. coli</i> cells will get a library of PobR
										mutants. Strains with PobR<sup>mut</sup> that recognize the aromatic compound
										are expected to grow in the LB agar medium.</p>
									<p>To further understand each part, please <a href="../project/design.html">click
											here</a> to view our design
										part.</p>
									<p>To constructed gYB2a-pobRWT-mCherry-codA-cmr, we first tried to
										generate plasmid gYb2a-PobR-P<sub>pobA</sub>*2-mcherry-SacB-Cmr by Goldengate
										assembly. However, when applying the plasmid
										gYb2a-PobR-P<sub>pobA</sub>*2-mcherry-SacB-Cmr to achieve functions as a dual
										screen system, the results are not ideal. </p>
									<p>We obtain the two target fragments, gYb2a-PobR-P<sub>pobA</sub>*2-mcherry-Cmr and
										CD by the method of PCR. Then the two fragments were ligated by using Goldengate
										assembly and transformed into <i>E. coli</i> BW&#916;<i>codA</i> competent cells and
										the plasmid gYB2a-pobRWT-mCherry-codA-cmr was constructed. </p>
									<p>The result of DNA sequencing showed that our
										gYB2a-pobRWT-mCherry-codA-cmr plasmid was constructed.</p>
								</div>
								<div id="slide1-chapter2">
									<h2>Directed evolution of PobR</h2>
									<p>Aiming to get strains that respond to different aromatic compounds by the dual
										selection system, a large PobR mutant library which is able to include as many
										mutants situations as possible is crucial. Therefore, we used error-prone PCR to
										construct the PobR mutant library. </p>
									<p>The generated library with highly random PobR mutants was transformed into
										<i>E.coli</i> BW&#916;codA to obtain transformants containing mutant plasmids.
										The PobR mutant library was transformed into BW&#916;codA competent cells and
										transferred to M9 medium for culturing in shaking flasks. Ten clones were
										randomly picked to sequence their PobR CDS regions for the quality control,
										which revealed diverse mutants with an average mutants rate of about 0.36%.
									<div class="figure-container auto-margin">
									</div>
								</div>
								<div id="slide1-chapter3">
									<h2>Fluorescence assay and screening of the PobR mutant library</h2>
									<p>Another part which is essential to get the new ligands specificities and
										detection range of PobR<sup>mut</sup> is the screening part.</p>
									<p>In the negative selection, the obtained strains were cultivated in liquid culture
										supplemented by 4HB and 5-FC. In the initial negative selection, we used a
										constant 4HB concentration of a relatively high level, 0.5 g/L, and then tested
										different concentrations of 5-FC to inhibit both the pseudo-positive and
										4HB-responsive strains. In this selection step, we first used 50 mg/L 5-FC, and
										observed insufficient inhibition of the bacterial growth. Thus, we increased the
										5-FC concentration to 200 mg/L, improving the selection effectiveness.</p>
									<p>In the positive selection, we added seven aromatic compounds to LB agar medium
										and cultivated <i>E. coli</i> cells harboring a library of PobR mutants in this
										medium for strain selection. As for the aromatic compounds, we chose
										phenylethanol (2-PE), mandelate (MA), 4-hydroxymandelate (HMA), phenylpyruvate
										(PPA), 4-hydroxyphenylpyruvate (HPP), phenylacetaldehyde (PAld) and p-Coumaric
										acid.</p>
									<p>After culturing for 30 hours, a few pink colonies were picked and cultured in a
										96 deep-well plate. Since the expression of the reporter gene <i>mCherry</i> was
										positively correlated with responsiveness, all that required is to transfer the
										pink colonies to LB media containing only Amp after activation and used 0.5 g/L
										of the test ligands for initial characterization. At the same time, negative
										control without the addition of test ligands was used to avoid a few biosensor
										variants with a strong fluorescence response in the absence of any ligand. After
										performing fluorescence measurement，we got some strains that are highly
										responsive to aromatic compounds. </p>
									<p>Besides, we used the dilution coating method to estimate the number of mutants
										which are capable of aromatic compounds per screen. The selection capacity for
										each compound was more than 900，000 clones (with at least four plates, the
										original density of the two-round selected bacteria was 450,000 CFU/mL.)</p>
									<p>Please <a href="../../static/protocol.zip">click here</a> to
										view our detailed protocol.</p>
								</div>
								<div id="slide1-chapter4">
									<h2>The specificity and detection range of PobR mutants responsive to new ligands
									</h2>
									<p>We obtained several responsive strains, of which the fluorescence intensity was
										1.5-fold higher than that of the negative control. To further evaluate the PobR
										mutants obtained above, the second round of characterization experiments were
										carried out to individually examine their responsiveness to each candidate
										ligand. For each ligand, we selected a mutant strain with the highest responsive
										profiles and plotted the curve for their ligand response. In total, 9 potential
										biosensors were isolated after the second round of characterization, and all
										these variants were sequenced to determine the mutations in their primary
										sequences. Amino acids at positions 163, 177 and 234 are located near the ligand
										binding pocket of PobR, and amino acid at position 40 is located in the DNA
										binding domain. </p>
								</div>
								<div id="slide1-chapter5" style="margin: 0 0 8%;">
									<h2>Modeling and docking</h2>
									<p>To better understand and analyze the effects of amino acid substitutions on the
										response of the PobR protein, we tried to use software to simulate the structure
										of the protein and build a molecular docking between the PobR protein and its
										inducer. </p>
									<p>We first use Homologous Model website SWISS-MODEL to construct the PobR mutant
										model using the PobR wild type as a template.</p>
									<p>Secondly, we obtained the structure files of the ligands 4HB, 2-PE, MA, HMA,
										PAld, HPP and PPA from the organic small molecule database Pubchem. For more
										details you can <a href="https://pubchem.ncbi.nlm.nih.gov/">click here</a>.</p>
									<p>Based on the structures of PobR monomer and ligands, the docking engine Autodock
										is used to simulate molecular docking. Autodock search space coordinates were
										set
										as center_x = -4.672, center_y = 3.331, center_z = -2.213. Dimensions of the
										search space were set as size_x = 40, size_y = 40, size_z = 40, and
										exhaustiveness was set at 15. The 15 conformational conditions in a score based
										on the lowest binding energy were listed as the docking results. To examine the
										accuracy of our docking, we used Ligplus to check the interaction between the
										small molecule and predicted protein receptors. Finally, the three-dimensional
										schematics of the protein and its ligand were portrayed using PyMol Version
										2.2.0.</p>
								</div>
							</div>
						</div>
					</section>
				</div>
			</div>
		</div>
		<a href="#" id="mobile-to-top-arrow" class="to-top">
			<p style="font-size: 20px;">TOP</p>
			<img style="max-height: 60px" src="../../static/home/top.png">
		</a>
		<!---------- 页脚FOOTER ------------>
		<footer>
			<div>
				<div class="c-footer">
					<div class="u-container">
						<div class="u-hide-print">
							<div class="c-footer__header">
							</div>
							<div class="c-footer__grid">
								<div class="c-footer__group">
									<h3 class="c-footer__heading">About us</h3>
									<ul class="c-footer__list">
										<li class="c-footer__item"><a href="#" data-track="click">
												This team is composed of enthusiastic students
												from three different colleges and various majors. To spy out the secrets
												of directed evolution, a group of beginners are looking forward to
												creating extraordinary experiences and we did it.</a></li>
									</ul>
								</div>
								<div class="c-footer__group">
									<h3 class="c-footer__heading">About topic</h3>
									<ul class="c-footer__list">
										<li class="c-footer__item"><a href="#" data-track="click"> A dual selection
												system for directed evolution to identify allosteric transcription
												factor PobR variants responsive to different aromatic compounds
											</a></li>
									</ul>
								</div>
								<div class="c-footer__group">
									<h3 class="c-footer__heading">Quicklinks</h3>
									<ul class="c-footer__list">
										<li class="c-footer__item"><a href="../project/background.html"
												data-track="click">Background</a></li>
										<li class="c-footer__item"><a href="../project/design.html"
												data-track="click">Design</a></li>
										<li class="c-footer__item"><a href="../project/results.html"
												data-track="click">Result</a></li>
									</ul>
								</div>
								<div class="c-footer__group">
									<h3 class="c-footer__heading">Contact info</h3>
									<ul class="c-footer__list">
										<li class="c-footer__item"><a href="#" data-track="click">
												Email : iDEC_NEFU_2022@163.com</a></li>
										<li class="c-footer__item"><a href="#" data-track="click">
												Address : Northeast
												Forestry University, Harbin,
												Heilongjiang 150040, P. R. China</a></li>
									</ul>
								</div>
							</div>
						</div>
					</div>
				</div>
			</div>
			<div class="c-corporate-footer">
				<div class="u-container">
					<img src="../../static/logo.png" width="120px" height="40px">
					<p class="c-corporate-footer__legal">iDEC-2022 NEFU_China</p>
				</div>
			</div>
		</footer>
		<script src="../../js/jQuery.js" type=text/javascript></script>
		<script src="../../js/detect.js" type="text/javascript"></script>
		<script src="../../js/main_javascript.js" type=text/javascript></script>
		<script src="../../js/MoveContent.js" type="text/javascript"></script>
		<script>
			document.addEventListener("DOMContentLoaded", function(event) {
				MoveContentOutside();
			});
		</script>
	</body>
</html>
