<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
		<title>iDEC NEFU_China 2022</title>
		<link rel="stylesheet" type="text/css" href="../../css/main_cssstyle.css">
		<style>
			header {
				background: url("../../static/top/results.png");
				min-height: 50%;
				background-repeat: no-repeat;
				background-size: cover;
			}
			.page-title {
				padding-top: 10em;
				padding-bottom: 1.5em;
			}

			/* footer的样式 */
			.contact-icon {
				display: flex;
				align-items: center;
				/*font-size: 3em;*/
				color: white;
			}

			.contact-icon a {
				color: white;
			}

			.contact-icon i {
				font-size: 3em;
			}
			.t{
				font-size: 0.8em;
			}
		</style>
	</head>
	<body>

		<a class="skip-main" href="#slide1-chapter1">Skip to main content</a>
		<header>
			<nav id="normal-nav">
				<!---- 常规导航栏Navigation bar (normal) -->
				<div class="logo">
					<img style="max-height: 22em" id="nav-logo" src="../../static/logo.png"
						alt="This is our logo, but if you click it, you will return home">
				</div>
				<div class="chapters">
					<div class="dropdown-menu cool-link">
						<a class="chapter-nav " href="https://idec-teams.github.io/2022_NEFU_China/">
							<div class="chapter-content">
								<div class="chapter-title active ">
									Home
								</div>
							</div>
						</a>
					</div>
					<div class="dropdown-menu cool-link">
						<a class="chapter-nav " href="#">
							<div class="chapter-content">
								<div class="chapter-title active ">
									Team
								</div>
								<div class="chapter-icon">
									<i class="fas fa-sort-down"></i>
								</div>
							</div>
						</a>
						<div class="dropdown-content">
							<a href="../team/members.html" class="dropdown-page cool-link2">
								Members
							</a>
							<a href="../team/attributions.html" class="dropdown-page cool-link2">
								Attributions
							</a>
						</div>
					</div>
					<div class="dropdown-menu cool-link">
						<a class="chapter-nav " href="#">
							<div class="chapter-content">
								<div class="chapter-title active ">
									Project
								</div>
								<div class="chapter-icon">
									<i class="fas fa-sort-down"></i>
								</div>
							</div>
						</a>
						<div class="dropdown-content">
							<a href="../project/background.html" class="dropdown-page cool-link2">
								Background
							</a>
							<a href="../project/design.html" class="dropdown-page cool-link2">
								Design
							</a>
							<a href="../project/results.html" class="dropdown-page cool-link2">
								Results
							</a>
							<a href="../../static/report.pdf" class="dropdown-page cool-link2" target="_blank">
								Report
							</a>
							<a href="../../static/supplementary information.pdf" class="dropdown-page cool-link2" target="_blank">
								Supplementary information
							</a>
						</div>
					</div>
					<div class="dropdown-menu cool-link">
						<a class="chapter-nav " href="#">
							<div class="chapter-content">
								<div class="chapter-title active ">
									Documentation
								</div>
								<div class="chapter-icon">
									<i class="fas fa-sort-down"></i>
								</div>
							</div>
						</a>
						<div class="dropdown-content">
							<a href="../documentation/experiments.html" class="dropdown-page cool-link2">
								Experiments
							</a>
							<a href="../documentation/notebook.html" class="dropdown-page cool-link2">
								Notebook
							</a>
							<a href="../documentation/safety.html" class="dropdown-page cool-link2">
								Safety
							</a>
							<a href="../project/protocol.html" class="dropdown-page cool-link2">
								Protocol
							</a>
						</div>
					</div>
				</div>
			</nav>
			<!------------ 页面标题PAGE TITLE -------------------->
			<div class="page-title">
				<div class="page-title-content">
					<h1>RESULTS</h1>
					<!---Page title -->
				</div>
			</div>
		</header>
		<div class="grid-normal">
			<div class="introduction grid-normal">
			</div>
		</div>
		<div style="display:none" id="marker" class="triangle-down"> </div>
		<!------------------ BUTTONS ---------------------------->
		<div class="buttons">
		</div>
		<div class="sections-container">
			<aside class="sticky">
				<div id="sub-nav" class="sub-navigation-content">
					<div class="sub-navigation-chapter cool-link2">
						<a class="main-nav-link" linkedidchapter="slide1-chapter1">Objectives</a>
						<!------Chapter one -->
					</div>
					<div class="sub-navigation-chapter cool-link2">
						<a class="main-nav-link" linkedidchapter="slide1-chapter2">Results</a>
						<!------Chapter Two -->
					</div>
					<div class="sub-navigation-chapter cool-link2">
						<a class="main-nav-link" linkedidchapter="slide1-chapter3">
							<li class="t">Construction of biosensors</li>
						</a>
						<!------Chapter Three -->
					</div>
					<div class="sub-navigation-chapter cool-link2">
						<a class="main-nav-link" linkedidchapter="slide1-chapter4">
							<li class="t">Construction of mutant libraries</li>
						</a>
						<!------Chapter Three -->
					</div>
					<div class="sub-navigation-chapter cool-link2">
						<a class="main-nav-link" linkedidchapter="slide1-chapter5">
							<li class="t">Outcomes of the dual selections</li>
						</a>
						<!------Chapter Three -->
					</div>
					<div class="sub-navigation-chapter cool-link2">
						<a class="main-nav-link" linkedidchapter="slide1-chapter6">
							<li class="t">Structure modeling and ligand
								docking</li>
						</a>
						<!------Chapter Four -->
					</div>
			
					<!--            返回顶部箭头-->
					<div id="to-top-arrow">
						<a title="Back to top" href="#"><i class="fas fa-chevron-circle-up"></i></a>
						<!------Chapter Three -->
					</div>
				</div>
			</aside>
			<div class="carousel">
				<div class="slider">
					<!------------------------------ section 1------------------------------------>
					<section class="slide-sections">
						<div class="main-body">
							<div class="main-body-content">
								<div id="slide1-chapter1">
									<h2>Objectives</h2>
									<p>The objective of our study was to develop a robust dual selection system used to
										identify PobR mutants responsive to a series of aromatic compounds. We first
										built a PobR mutant library that was established through random mutagenesis and
										included 21,000 mutants to evaluate the effectiveness of our dual selection
										system.</p>
								</div>
								<div id="slide1-chapter2">
									<h2>Results</h2>
									<p>
										<b id="slide1-chapter3">Construction of biosensors</b>
									</p>
									<p>Based on the aforementioned design, we placed the <i>mCherry</i> (a red
										fluorescent
										protein) gene, cytosine deaminase (<i>codA</i>) gene, and chloramphenicol
										resistant
										(<i>cmr</i>) gene under the control of the <i>pobR</i> promoter
										(P<sub>pobA</sub>) to create the dual
										selection system (Figure 1).
										In the expression vector, the <i>pobR</i> gene is controlled by a constitutive
										promoter, and the genes responsible for the selection are driven by the
										P<sub>pobA</sub>.
										The consensus element (Oi) can bind PobR leading to repressed expression of the
										selective genes. When the PobR<sup>WT</sup> is allosterically activated by its
										native
										ligand 4HB and subsequently triggers the P<sub>pobA</sub> promoter, CDase is
										produced and
										converts exogenously added non-toxic 5-FC to toxic 5-FU, representing a negative
										selection. When the P<sub>pobA</sub> is triggered by a PobR mutant that is allosterically
										activated by a tested aromatic compound in the absence of 5-FC, the bacteria
										become Cm-resistant.</p>
									<div class="figure-container" style="width: 60%;margin: 0 auto;"">
										<img src=" ../../static/result/fig1.jpg" alt="">
									</div>
									<p><b>Figure 1.</b> Schematic diagram of the dual selection system.</p>
									<p>Thereafter, we verified the feasibility of the dual selection system using the expression vector containing gYB2a-pobR<sup>WT</sup>-mCherry-codA-cmr, the
										PobR<sup>WT</sup> biosensor. Before
										starting, we transferred the PobR<sup>WT</sup> biosensor into the <i>E. coli</i>
										BW25113 strain,
										in which the <i>codA</i> gene was knocked out (BW&#916;<i>codA</i>). In the
										presence of 0.5 g/L 4HB,
										the PobR<sup>WT</sup> biosensor strain was resistant to chloramphenicol, and the
										growth of
										the bacteria was significantly better than that without 4HB. In experiments
										simulating the negative selection, the strains grew normally in the presence of
										either 4HB or 5-FC alone, but the bacterial growth was inhibited with both
										chemicals present in the medium (Figure 2).</p>
									<div class="figure-container" style="width: 90%;margin: 0 auto;">
										<img src=" ../../static/result/fig2.png" alt="">
									</div>
									<p><b>Figure 2.</b> Model experiments of negative and positive selections using
										the dual selection system. The OD<sub>600</sub> was measured after 12 h of
										cultivation.The criterion of statistical significance level was denoted as follows: *P < 0.05; **P < 0.01; ***P < 0.001.</p>
								</div>
								<div id="slide1-chapter4" style="margin-bottom:10%;">
									<p>
										<b>Construction of mutant libraries</b>
									</p>
									<p>We designed primers with special interfaces, and obtained a large number of PobR
										mutant fragments by error-prone PCR, which were subsequently inserted into the
										vector harboring the P<sub>pobA</sub>, mCherry, codA and <i>cmr</i>,
										through the Golden Gate
										Assembly system, and finally transformed into the BW&#916;<i>codA</i> strain. As
										a result, we
										constructed a random mutagenesis PobR library with a storage capacity of 21,000
										clones and an average mutation rate of 0.36%.</p>
									<div class="figure-container" style="width: 100%;margin: 0 auto;"">
										<img src=" ../../static/result/fig3.jpg" alt="">
									</div>
									<p><b>Figure 3.</b> Schematic diagram of constructing the PobR mutant library.</p>
									<p id="slide1-chapter5">
										<b>Outcomes of the dual selections</b>
									</p>
									<div class="figure-container" style="width: 90%;margin: 0 auto;"">
										<img src=" ../../static/result/fig4.jpg" alt="">
									</div>
									<p><b>Figure 4.</b> Flow chart of the dual selection process</p>
									<p>As shown in Figure 4, we performed two rounds of the negative selection of the
										PobR mutants using 50 mg/L and 200 mg/L of 5-FC. By raising 5-FC concentration,
										we successfully eliminated pseudo-positive strains and those highly responsive
										to 4HB, increasing the effectiveness of the selection from 89% to 98%.
										In the positive selection, bacterial cultivation was conducted using the
										negative selection liquid medium individually supplied with different aromatic
										compounds, as well as chloramphenicol. Before starting, we evaluated the volume
										of the forward screening. The experimental procedure included aspirating 50 µl
										of the selected bacterial solution, adding it into 5,000 µl of LB medium (i.e.,
										1:100 dilution), and then taking 50 µl of the diluted bacteria to spread onto a
										LB-agar plate. Based on our calculation, the original density of the two-round
										selected bacteria was 450,000 CFU/mL. At least four plates were screened for
										each compound, with approximately 225,000 raw clones per plate.</p>
									<div class="figure-container" style="width: 60%;margin: 0 auto;"">
										<img src=" ../../static/result/fig5.jpg" alt="">
									</div>
									<p><b>Figure 5.</b> Clone calculation after passing two rounds of the negative
										selection in medium containing 50 mg/L and 200 mg/L 5-FC using the dilution
										coating method.</p>
									<p> After preliminary characterization with 0.5 g/L of candidate ligands, we
										obtained several ligand-responsive strains, in which the fluorescence intensity
										was at least 1.5-fold higher than that of the negative control. Subsequently, we
										performed a second round of characterization experiments to individually examine
										their reactivity to each candidate ligand. After the second round of
										characterization, a total of nine potential PobR mutant biosensors were
										isolated, and their ligand response curves were plotted (Figure 6). Among them,
										the PobR mutants with relatively strong responses to HPP or PAA were selected.
									<div class="figure-container" style="width: 90%;margin: 0 auto;"">
										<img src=" ../../static/result/fig6.png" alt="">
									</div>
									<p><b>Figure 6.</b> Characterization of the PobR mutants responsive to different
										aromatic compounds. (a) to (i). The fluorescent gene expression normalized by
										cell density (RFU/OD<sub>600</sub>) was plotted.</p>
									</p>
									<p>All nine potential biosensors obtained above were sequenced to identify the
										mutations in their primary sequences(Figure 7).  </p>
									<div class="figure-container" style="width: 95%;margin: 0 auto;"">
										<img src=" ../../static/result/fig7.png" alt="">
									</div>
									<p><b>Figure 7.</b> Amino acid substitutions in the transcriptional factor PobR
										mutants as biosensor candidates.</p>
									<p>Analysis of the spatial locations of the mutated amino acids revealed that the
										residues 163, 177 and 234 of PobR are located near the ligand binding pocket,
										and the residue 40 is located in the DNA binding domain. The mutations of these
										amino acids likely exerted large impacts on the ligand recognition specificity
										of the PobR protein (Figure 8).</p>
									<p>The remaining mutated residues (S114, Y145, L183, L198, L201, L202, V225, Q250
										and V255) are distant from both the ligand binding and DNA binding domains of
										PobR, and thus the reasons for their altered binding affinity remain unclear. We
										speculated that the amino acid changes at these sites might indirectly alter the
										conformation of the ligand binding domain, or affect the dimerization of PobR.
										Interestingly, we discovered that four of the twelve variants harbored the same
										point mutation of S114A, indicating that the S-to-A substitution at the residue
										144 could affect PobR ligand specificity to a great extent.</p>
									<div class="figure-container" style="width: 60%;margin: 0 auto;"">
										<img src=" ../../static/result/fig8.png" alt="">
									</div>
									<p><b>Figure 8.</b> Analysis of the positions of mutated amino acids in PobR
										mutants using the PyMOL.</p>
									<p>Among these PobR mutants, several of them were favorably responsive to HPP. As
										HPP concentration raised from 0.01 g/L to 1 g/L, their mCherry fluorescence
										showed a significant increase after 12 h of incubation. Among them, the best
										responsive sensor was the PobR mutant clone 0914-A8-1 with a point mutation of
										W177R, which showed a 5.1-fold increase of mCherry fluorescence signal in
										response to HPP (Figure 6). PobR<sup>W177R</sup> showed a significant
										change in specificity
										and a wider dynamic range (0.01-1.0 g/L), indicating decreased ligand binding
										affinity. We evaluated the specificity of PobR<sup>W177R</sup> to detect
										different aromatic
										compounds, and observed that 4HB could still activate it to enhance mCherry
										expression. Notably, although the PobR<sup>W177R</sup> mutant retained a
										5.5-fold response
										to 4HB versus the condition without any ligand, its affinity to 4HB was
										significantly lower than that of PobR<sup>WT</sup> (28-fold reduction).</p>
									<p>Additionally, at 0.5 g/L of PPA, another clone, PobR<sup>R40C</sup>, exhibited
										over 2 times
										of maximal induction by PPA compared to the condition without the ligand.
										Importantly, this PobR mutant showed marginal basal transcriptional activity.
										However, we also tested multiple additional aromatic compounds, including HMA,
										MA, p-Coumarins, etc., in the positive selection, and observed relatively weak
										responses to these compounds by some mutants. This could be due to the lack of
										capacity of our mutant library and insufficient selection rounds for these
										compounds. </p>
										<p>
											<b id="slide1-chapter6">Structure modeling and ligand docking</b>
										</p>
									<p>We used the SWISS-MODEL to simulate the three-dimensional structure of
										PobR<sup>W177R</sup>,
										and Autodock to simulate the interactions of HPP to PobR<sup>W177R</sup> and
										PobR<sup>WT</sup>. After
										analyzing the docking results, we found that HPP could form hydrogen bonds with
										I235 and R177 of PobR<sup>W177R</sup>, and hydrophobic interactions with L169,
										L165, L122,
										and V121 of PobR<sup>WT</sup>. In the surface model, R177 is spatially close to
										the HPP
										binding pocket of PobR<sup>W177R</sup>, while W177 is likely closer to a α-helix
										in PobR<sup>WT</sup>
										than that in the PobR<sup>W177R</sup> mutant. A comparison of hydrogen bonds
										revealed that
										R177 formed a hydrogen bond with HPP in the mutant, while W177 showed no
										interaction with HPP in PobR<sup>WT</sup>. Based on these analyses, we concluded
										that the
										W177R substitution allowed HPP to better interact with PobR.</p>
									<div class="figure-container" style="width: 60%; margin: 0 auto;">
										<img src="../../static/result/fig9.png" alt="">
									</div>
									<p><b>Figure 9.</b> Structural analyses of PobR<sup>W177R</sup> and
										PobR<sup>WT</sup> proteins using the
										Autodock and PyMOL. Docking analyses generate binding simulation
										between HPP and the residues of PobR proteins.</p>
								</div>
							</div>
						</div>
					</section>
				</div>
			</div>
		</div>
		<a href="#" id="mobile-to-top-arrow" class="to-top">
			<p style="font-size: 20px;">TOP</p>
			<img style="max-height: 60px" src="../../static/home/top.png">
		</a>
		<!---------- 页脚FOOTER ------------>
		<footer>
			<div>
				<div class="c-footer">
					<div class="u-container">
						<div class="u-hide-print">
							<div class="c-footer__header">
							</div>
							<div class="c-footer__grid">
								<div class="c-footer__group">
									<h3 class="c-footer__heading">About us</h3>
									<ul class="c-footer__list">
										<li class="c-footer__item"><a href="#" data-track="click">
												This team is composed of enthusiastic students
												from three different colleges and various majors. To spy out the secrets
												of directed evolution, a group of beginners are looking forward to
												creating extraordinary experiences and we did it.</a></li>
									</ul>
								</div>
								<div class="c-footer__group">
									<h3 class="c-footer__heading">About topic</h3>
									<ul class="c-footer__list">
										<li class="c-footer__item"><a href="#" data-track="click"> A dual selection
												system for directed evolution to identify allosteric transcription
												factor PobR variants responsive to different aromatic compounds
											</a></li>
									</ul>
								</div>
								<div class="c-footer__group">
									<h3 class="c-footer__heading">Quicklinks</h3>
									<ul class="c-footer__list">
										<li class="c-footer__item"><a href="../project/background.html"
												data-track="click">Background</a></li>
										<li class="c-footer__item"><a href="../project/design.html"
												data-track="click">Design</a></li>
										<li class="c-footer__item"><a href="../project/results.html"
												data-track="click">Result</a></li>
									</ul>
								</div>
								<div class="c-footer__group">
									<h3 class="c-footer__heading">Contact info</h3>
									<ul class="c-footer__list">
										<li class="c-footer__item"><a href="#" data-track="click">
												Email : iDEC_NEFU_2022@163.com</a></li>
										<li class="c-footer__item"><a href="#" data-track="click">
												Address : Northeast
												Forestry University, Harbin,
												Heilongjiang 150040, P. R. China</a></li>
									</ul>
								</div>
							</div>
						</div>
					</div>
				</div>
			</div>
			<div class="c-corporate-footer">
				<div class="u-container">
					<img src="../../static/logo.png" width="120px" height="40px">
					<p class="c-corporate-footer__legal">iDEC-2022 NEFU_China</p>
				</div>
			</div>
		</footer>

		<script src="../../js/jQuery.js" type=text/javascript></script>
		<script src="../../js/detect.js" type="text/javascript"></script>
		<script src="../../js/main_javascript.js" type=text/javascript></script>
		<script src="../../js/MoveContent.js" type="text/javascript"></script>
		<script src="https://kit.fontawesome.com/50c1a3d91f.js" crossorigin="anonymous"></script>

		<script>
			document.addEventListener("DOMContentLoaded", function(event) {
				MoveContentOutside();
			});
		</script>
	</body>
</html>
